slide/src/test/ui/distribute_negation.slide

```diff
@@ -0,0 +1,15 @@
+===in
+-(a - b)
+===in
+
+~~~stdout
+~~~stdout
+
+~~~stderr
+error: Found argument '-(' which wasn't expected, or isn't valid in this context
+
+USAGE:
+    slide [FLAGS] [OPTIONS] <program>
+
+For more information try --help
+~~~stderr
```

Comment:
Ahh this is a bug

Reply:
Going forward, I'm planning to add two more options:
- `~~~exitcode`
- `@TODO: ...` as a directive, with an option to fail on TODO tests